content/reference-transaction-format.md

```diff
@@ -476,6 +479,107 @@ In `rippled`'s WebSocket and JSON-RPC APIs, the TransferRate is represented as a
+| `FinishAfter`    | Number    | UInt32            | (Optional) The time, in [seconds since the Ripple Epoch](reference-rippled.html#specifying-time), when the escrowed XRP can be released to the recipient. This value is immutable; the funds cannot move until this time is reached. |
+| `Condition`      | String    | VariableLength    | (Optional) Hex value representing a [PREIMAGE-SHA-256](https://tools.ietf.org/html/draft-thomas-crypto-conditions-02#section-8.1) [crypto-conditions](https://tools.ietf.org/html/draft-thomas-crypto-conditions-02). The funds can only be delivered to the recipient if this condition is fulfilled. |
+| `DestinationTag` | Number    | UInt32            | (Optional) Arbitrary tag to further specify the destination for this escrowed payment, such as a hosted recipient at the destination address. |
+| `SourceTag`      | Number    | UInt32            | (Optional) Arbitrary tag to further specify the source for this escrowed payment, such as a hosted sender at the source address. |
+
+Either `CancelAfter` or `FinishAfter` must be specified. If both are included, the `FinishAfter` time must precede that of `CancelAfter`.
+
+
+
+## EscrowFinish ##
+
+[[Source]<br>](https://github.com/ripple/rippled/blob/develop/src/ripple/app/tx/impl/Escrow.cpp "Source")
+
+_Requires the [Escrow Amendment](concept-amendments.html#escrow)._
+
+Deliver escrowed XRP the recipient.
+
+Example EscrowFinish:
+
+```json
+{
+    "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+    "TransactionType": "EscrowFinish",
+    "Owner": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+    "OwnerSequence": 7,
+    "Condition": "A0258020E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855810100",
+    "Fulfillment": "A0028000"
+}
+```
+
+| Field           | JSON Type        | [Internal Type][] | Description               |
+|:----------------|:-----------------|:------------------|:--------------------------|
+| `Owner`         | String           | AccountID         | Address of the source account that funded the escrow payment.
+| `OwnerSequence` | Unsigned Integer | UInt32            | Transaction sequence of [EscrowCreate transaction](#escrowcreate) that created the escrow to finish.
+| `Condition`     | String           | VariableLength    | (Optional) Hex value representing a [PREIMAGE-SHA-256](https://tools.ietf.org/html/draft-thomas-crypto-conditions-02#section-8.1) [crypto-conditions](https://tools.ietf.org/html/draft-thomas-crypto-conditions-02). This must match the `Condition` specified in the [EscrowCreate transaction](#escrowcreate). |
+| `Fulfillment`   | String           | VariableLength    | (Optional) Hex value of the `Condition` preimage |
+
+Any account may submit an EscrowFinish transaction.
+If the corresponding [EscrowCreate transaction](#escrowcreate) specified a `FinishAfter` time that is after the close time of the most recently-closed ledger, the EscrowFinish transaction fails.
```

Comment:
Add a blank line before this line, and make this a bullet point. (Otherwise, Markdown just treats it as a hard-wrapped continuation of the previous line.)